config/application.yml.tt

```diff
@@ -6,6 +6,9 @@ default: &default
   MAILER_DEFAULT_HOST: "localhost"
   MAILER_DEFAULT_PORT: "3000"
   MAILER_SENDER: "Test <noreply@nimblehq.co>"
+  AVAILABLE_LOCALES: "en"
```

Comment:
@andyduong1920 I assume there will be a bit confusing here:
Should we add one more locale as an example: `"en, th"`?

Because if the user does not add the space then the locale will break, as the _parsing locale config_ code in https://github.com/nimblehq/rails-templates/pull/241/files#r503012228

Reply:
@byhbt I think that doesn't make sense to add more locale at default as this one is the template and not all projects need th locale, so instead of adding more example here, I think we can adding the comment in the https://github.com/nimblehq/rails-templates/pull/241/files#diff-da60b4e96eff2b132991226d308949e23f4ef3aad45ad59edd09cbc32cc6251eR24

Reply:
yes, that's fine also.